css/login.css

```diff
@@ -0,0 +1,29 @@
+ody {
+    box-sizing: border-box;
+}
+
+#external {
+    background-image:url(../assets/feedback-banner.jpg);
+        
+}
+
+img {
+/*    float:left;*/
+    height: 20rem;
+    width: 20rem;
+}
+
+
+#fb1 {
```

Comment:
What is the difference between `#fb1` and `#fb2`? Would there be any way to bundle them up together so we don't have to repeat the same code twice?